src/datasets/utils/resources/tasks.json

```
@@ -98,6 +98,43 @@
             "univariate-time-series-forecasting"
         ]
     },
+    "object-detection": {
+        "description": "tasks related to detecting instances of objects from a particular class in an image",
+        "options": [
+            "face-detection",
+            "other"
+        ]
+    },
+    "image-to-text": {
+        "description": "tasks related to generating images from an existing dataset",
```

maybe this is clearer?
```suggestion
        "description": "tasks related to generating images from a given text",
```

---

Actually, this is the other way around I think. This should be generating text from a given image